src/libs/Clipboard/index.ts

```diff
@@ -39,17 +57,17 @@ function setHTMLSync(html, text) {
         };
     } else {
         originalSelection = {
-            anchorNode: selection.anchorNode,
-            anchorOffset: selection.anchorOffset,
-            focusNode: selection.focusNode,
-            focusOffset: selection.focusOffset,
+            anchorNode: selection?.anchorNode,
+            anchorOffset: selection?.anchorOffset,
+            focusNode: selection?.focusNode,
+            focusOffset: selection?.focusOffset,
         };
     }
 
-    selection.removeAllRanges();
+    selection?.removeAllRanges();
     const range = document.createRange();
     range.selectNodeContents(node);
-    selection.addRange(range);
+    selection?.addRange(range);
 
```

Comment:
If we cannot guarantee `selection` isn't null, can we just return early from this function? Rather than have to make every following call optional 

_I'm asking this based on Swift/Kotlin, so maybe the guidelines are different for Typescript_.

Reply:
@fabioh8010 any thoughts here?

Reply:
@Julesssss AFFAIK [optional chaining](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#ts-nullish-coalescing) can work in cases like this. But I tried your suggestion and it looks great too.

Reply:
Oh I'm all for optional chaining, it's just in this case it seems a bit overkill. We could be iterating through all these lines without anything being invoked, which seems a bit unclean. 

Can we do something like this with Typescript? 

```
e.clipboardData?.let {
    e.clipboardData.clearData();
    e.clipboardData.setData('text/html', html);
    e.clipboardData.setData('text/plain', text);
}
```